react/features/toolbox/components/web/Toolbox.js

```diff
@@ -17,7 +17,10 @@ import {
 } from '../../../base/participants';
 import { connect } from '../../../base/redux';
 import { OverflowMenuItem } from '../../../base/toolbox';
-import { getLocalVideoTrack, toggleScreensharing } from '../../../base/tracks';
```

Comment:
Why do we need this change?

Reply:
Not needed it is left over from previous conference.js version, when i had another function to toggle blur effect

Reply:
If this is the case could you please revert this?